app/_src/mesh/support-policy.md

```diff
@@ -11,7 +11,9 @@ The support for {{site.mesh_product_name}} software versions is explained in thi
 
 | Version  | Released Date | End of Full Support |
 |:--------:|:-------------:|:-------------------:|
+|  2.1.x   |  2023-02-01   |     2025-02-01      |
 |  2.0.x   |  2022-11-04   |     2024-11-03      |
```

Comment:
@slonka Could you update 2.0 and 2.1 so the support is for only one year?

Reply:
yeah, sorry for the late reply but I was on duty and had a ton of stuff to work on, thanks! 👍